components/NotificationBar.js

```diff
@@ -1,21 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import styled from 'styled-components';
+
+import AcceptRejectButtons from './host-dashboard/AcceptRejectButtons';
 
 const logo = '/static/images/opencollective-icon.svg';
 
+const AcceptRejectBanner = styled.div`
+  display: flex;
+  font-size: 1em;
+  width: 300px;
```

Comment:
I don't think setting a `width` is required, it's currently creating centering issues and may cause troubles with internationalization because the translated strings can be longer or shorter than their base. I suggest:

```suggestion
  flex-wrap: wrap;
  justify-content: center;
```